docs/Developers/Design/CRD_design.md

```diff
@@ -0,0 +1,69 @@
+This document describes the set of [Custom Resource Definitions
+(CRDs)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+that will be used to deploy mixins for a storage system. The actual implementation
```

Comment:
~that will be used to deploy mixins for a storage system. The actual implementation
of the resources described here will be phased in during development. The
purpose of this document is to provide the overall structure, ensuring the end
result provides necessary configurability in a user-friendly manner.~

`that will be used to deploy mixin and ensure user-friendly configuration options  for monitoring storage systems.`

Reply:
Ack